src/main/kotlin/mate/academy/LetterRemover.kt

```diff
@@ -1,5 +1,10 @@
 package mate.academy
 
-class LetterRemover {
-    // implement ...
+object LetterRemover {
+    @JvmStatic
+    fun removeLetter(input: String): CharArray {
```

Comment:
FYI could be single-lined
smth like this
fun removeLetter(input: String) = input.lowercase().replace('r', '!').toCharArray()